p4-fusion/utils/p4_helpers.h

```diff
@@ -0,0 +1,3 @@
+#include <string>
+
+std::string decodePath(std::string input);

```

Comment:
I think the input string can be a const ref?
```suggestion
std::string decodePath(const std::string& input);

```

(is this the right syntax?)